textMining.py

```diff
@@ -0,0 +1,111 @@
+import wikipedia
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import requests
+
+def wikiInfo():
+    """
+        Originally worked with wikipedia to look at online text
+
+        returns: tuple of text strings from wikipedia articles
+    """
+
+    # finds the pages
+    cat = wikipedia.page("Cat")
+    dog = wikipedia.page("Dog")
+
+    # saves the content of the pages as strings
+    catInfo = cat.content
+    dogInfo = dog.content
+
+    # returns tuple of the full text strings
+    return catInfo, dogInfo
+
```

Comment:
It seems like your wikipedia scrapping code is not a part of your final analysis. In this case, it is unnecessary to include this function. Shorter code without any unnecessary parts are more readable and understandable.